pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -129,6 +130,7 @@
 
   private final int _defaultHllLog2m;
   private final boolean _enableQueryLimitOverride;
+  private final boolean _enableSegmentPartitionedDistinctCountOverride;
```

Comment:
I don't think we need a config for this. We should always override if the distinctcount column is segment partitioned